cryptoapi_windows.go

```diff
@@ -9,16 +9,60 @@ import (
 	"time"
 
 	"golang.org/x/sys/windows/registry"
+	"gopkg.in/hlandau/easyconfig.v1/cflag"
+)
+
+var (
+	cryptoApiFlagGroup            = cflag.NewGroup(flagGroup, "capi")
+	cryptoApiFlagLogicalStoreName = cflag.String(cryptoApiFlagGroup, "store", "Root",
+		"Name of CryptoAPI logical store to inject certificate into. Consider: Root, Trust, CA, My")
+	cryptoApiFlagLogicalStoreScope = cflag.String(cryptoApiFlagGroup, "scope", "HKLM",
+		"Scope of CryptoAPI certificate store. Valid choices: HKLM, HKCU, machine, user")
 )
 
 // In 64-bit Windows, this key is shared between 64-bit and 32-bit applications.
 // See https://msdn.microsoft.com/en-us/library/windows/desktop/aa384253.aspx
-const cryptoApiCertStoreRegistryBase = registry.LOCAL_MACHINE
-const cryptoApiCertStoreRegistryKey = `SOFTWARE\Microsoft\EnterpriseCertificates\Root\Certificates`
```

Comment:
This PR removes the previously existing functionality of writing to the `Enterprise` physical store.  Would be good to restore that functionality.

Reply:
If I understand correctly, this is resolved by adding support for `-scope enterprise` in e3f4a1b